src/main/java/mate/jdbc/dao/CarDao.java

```diff
@@ -0,0 +1,20 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Car;
+
+public interface CarDao {
+
+    Car create(Car car);
+
+    Car get(Long id);
```

Comment:
when you get from dao layer it's a good practice to return Optional<> instead of simple class and check for nullable in service layer 
```suggestion
    Optional<Car> get(Long id);
```